# LQT Parameters

# Reference
Am:
  rows: 5
  cols: 5
  data: [0.999992870110446, -0.000712988288670474, -5.41441701278375e-08, -5.41441057874235e-06, 0.0,
         0.0199999524673852, 0.999992870110446, -3.60961305760057e-10, -5.41441701278375e-08, 0.0,
         0.000199999762336881, 0.0199999524673852, 0.999999999998195, -3.60961305760057e-10, 0.0,
         1.33333238268076e-06, 0.000199999762336881, 0.0199999999999928, 0.999999999998195, 0.0,
         6.66666349782453e-09, 1.33333238268076e-06, 0.000199999999999976, 0.0199999999999928, 1.0]

Bm:
  rows: 5
  cols: 1
  data: [0.0, 0.0, 0.0, 0.0, 0.0]

Cmx:
  rows: 1
  cols: 5
  data: [9.0, 0.0, 0.2166, 0.0, 0.0005]

Cmy:
  rows: 1
  cols: 5
  data: [2.0000, -0.2095, 0.0713, -0.0095, 0.0005] 

# Plant
# kp = 1.3
Ad:
  rows: 2
  cols: 2
  data: [1.0000, 0.0197,
         0.0,   0.9743]

Bd:
  rows: 2
  cols: 1
  data: [0.0003, 0.0257]

Cd:
  rows: 1
  cols: 2
  data: [1.0, 0.0]

## Initial Conditions
# Reference
xm: [1.0, 0.0, 0.0, 0.0, 0.0]

# Plant
x0x: [9.0, 0.0]
x0y: [2.0, 0.0]

## Gains

Kx:
  rows: 1
  cols: 7
  data: [0.545080178892607, 0.298244493046121, -4.83374763212812,
         0.0734074866054616, -0.116774466701902,
         0.00133785300048387, -0.000295129490099704]

Ky:
  rows: 1
  cols: 7
  data: [0.544832357786152, 0.297718734955302, -0.938787947829030,
         0.115468303660034, -0.0321069658667400,
         0.00512389590248099, -0.000294995309258785]


    ## // Kx << 27.124426673138792, 5.368769229468123, -244.03576687339714, 0.6608225415608986, -5.8735890982539685, 0.012275805209641881, -0.01356221333656942; 
    ## // Kx << 27.1244266731466,	5.36876922946919,	-244.035798047837,	0.660572823817245,	-5.87468455619007,	0.0120130717500125,	-0.0146863131760855;
    ## // Ky << 27.1244266731349,	5.36876922946723,	-52.9208135832513,	5.68328558567109,   -1.87385859124528,  0.257034874578231,	-0.0146863131760852;
    

# =======================
# Constants
# =======================
kp: 4
ki: 0.8
Qe: 1
R: 1                  
K0factor: 0.02
